wiki/Tournaments/GTS/IGTS_2021/en.md

```diff
@@ -0,0 +1,639 @@
+
+### Match regulations
+
+1. A private lobby will be created 10 minutes before the match started by a referee. Both team's captains will be invited into the room when it's match time. Then the referee or the captains will invite the other players into the room.
+2. If no staff or referee is available the match will be postponed.
+3. If less than the minimum required players attend, the maximum time the match can be postponed is **10 minutes**.
+4. There is no penalty for not joining the room instantly after match time as long as it's within the postponed time.
+5. If neither of both teams have the minimum required players after 10 minutes it will count as a loss for both teams in Group Stage. In the double elimination bracket the highest seeded team will advance.
+6. The match will be played as TeamVs and ScoreV2 and the name of the room must be: "IGTS 2021: (Team Red) vs (Team Blue)".
+7. Each team is allowed one warm-up but it must be **below 3 minutes**, and must be sent to the referee and streamer of the match one hour prior to the match's start time. Otherwise, the team will have no warmups.
+8. If a team does not respect the time said in the rule above, they will not have any warm-up for their match.
+9. Each captain has to ban **one beatmap** from the pool (apart from Tiebreaker). These beatmaps are not allowed to be picked by any team in the entire match.
+10. The captain of each team will roll with the command `!roll` in the chat.
+11. The team with the highest roll **bans second and picks first**. The team with the lowest roll **bans first and picks second**.
+12. Each captain has 2 mins to announce their ban. If they didn't tell the referee by that time the ban will be made with a roll.
+13. Beatmap selection will then alternate between each captain selecting a beatmap out of the map pool.
+14. Captains may pick freely from any bracket.
+15. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a team didn't tell their pick before the end of the timer the map will be picked with a roll.
+16. Exchanging players during a match is allowed without limitations.
+17. If a player disconnects between the beatmaps and the team cannot provide an exchange the match can be delayed 10 minutes at most. This delay can only be applied once.
+18. If a player disconnects, they get treated as if they failed the map.
+19. Disconnects within 30 seconds after the beatmap was started allows the map to be replayed. This is only allowed once per team.
+20. Failed players' scores do get added to the team score.
+21. If the beatmap ends in a draw the game will be nullified and the map will be replayed.
+22. In case of a tie in points (example: 4-4 in BO9) the tiebreaker is selected.
+
+### All Stars rules
+
+1. All Stars is a fun event taking place on the same weekend as the Grand Finals! It'll showcase the best players of the tournament playing against each other in a special match.
```

Comment:
```suggestion
1. All Stars is a fun event taking place on the same weekend as the Grand Finals! It will showcase the best players of the tournament playing against each other in a special match.
```